katsdpsigproc/tune.py

```diff
@@ -59,7 +58,11 @@
 _ScoreFunc = Callable[[int], float]
 _T = TypeVar("_T")
 
-KATSDPSIGPROC_TUNE_FALLBACK = os.getenv("KATSDPSIGPROC_TUNE_FALLBACK", "tune")
+KATSDPSIGPROC_TUNE_MATCH = os.getenv("KATSDPSIGPROC_TUNE_MATCH", "exact")
+if KATSDPSIGPROC_TUNE_MATCH not in ["exact", "nearest"]:
+    _logger.debug("KATSDPSIGPROC_TUNE_MATCH environment variable not one of [ exact | nearest ]: \
+                    setting to 'exact'.")
```

Comment:
If I'm not mistake the way you've wrapped that line means the string will have a big hunk of whitespace in the middle. A more common convention is to rely on Python's automatic concatenation given two separate strings.

It might be better to use `warnings.warn`, as logging probably won't have been properly configured at the time this module is imported.